src/gedcom-anonymizer/main.ml

```diff
@@ -10,5 +10,21 @@
 (*                                                                        *)
```

Comment:
It's better to keep all this in the `main.ml` file of the library. The idea is that the driver only contains the call to the main function, that is accessible from the library. This way, it is easy to create wrappers of the command inside another command.